layouts/partials/footer.hbs

```diff
@@ -1,3 +1,15 @@
+<a href="#" id="scrollToTop">&uarr; <span>Scroll To Top</span></a>
+
+<script type="text/javascript">
+    var $scrollToTop = document.getElementById('scrollToTop');
+    (window.onscroll = function() {
+        $scrollToTop.style.display = (window.pageYOffset > window.innerHeight) ? 'block' : 'none';
```

Comment:
In regards to my previous throttling question, the simplest fix for as little style changes as possible I could think of:
```js
var newVal = (window.pageYOffset > window.innerHeight) ? 'block' : 'none';
if (this.currentVal !== newVal) {
    $scrollToTop.style.display = newVal;
    this.currentVal = newVal;
}
```